src/test/data/ManualTesting/TestScript.md

```diff
@@ -0,0 +1,199 @@
+
+
+TestCase ID: TC001 <br>
+Title : Opening the help menu <br>
+Description : Tester should be able to open the help menu <br>
+Precondition : TaskTraker app is opened 
+
+Steps no. | Steps | Test Data |  ExpectedResult
+--- | :---------------- | :---------------- | :----------------
+1 | Go to the Command box on the top of the app | | User should be able to type anything here
+2 | Enter the help command in the box and press enter | help | a Help window should pop up
+3 | close the help window by pressing esc | | The help window should close
+
+
+TestCase ID: TC002 <br>
+Title : Adding a task, deadline, event <br>
+Description : Tester should be able to add a task event and deadline today <br>
+Format : <br>
+`add FLOATING_TASK_NAME [p-priority]` <br> 
+`add DEADLINE_NAME MM/DD/YYYY HH:MM[am/pm]` <br>
+`add EVENT_NAME MM/DD/YYYY HH:MM[am/pm] to MM/DD/YYYY HH:MM[am/pm]` <br>
+Format expanationation: <br>
+If the task name contains more than one words and spaces, please use `"name with space"`.<br>
+When indicating the time, only one of `am` and `pm` should be used.
+Pre-condition : TaskTraker Must be opened <br>
+Assumption : Tester is testing on 11-11-2016. If not, the event and deadline added will be marked as finishd and overdue respectively.
+
+Steps no. | Steps | Test Data |  ExpectedResult
+--- | :---------------- | :---------------- | :----------------
+1 | Go to the Command box on the top of the app | | User should be able to type anything here
+2 | Write/ Paste the Test Data into the box And press enter|  add "testing a program now" p-3 | The added task should be added and highlighted under 'Floating Task' and given a `Priority` of 3.
+3 | Repeat step 2 but with a different set of test data |  add "this is a deadline" 11/11/2016 11:15pm | the added deadline should be added and highlighted under 'Deadline'
+4 | Repeat step 2 one more time but adding a Event this time | add "test cs2103 product" 11/11/2016 4:15pm to 8:20pm | the added event should be added and highlighted under "Event"
+
+TestCase ID: TC003 <br>
+Title : The Flexibility of adding a task,event ,deadline <br>
+Description : Tester should be able to add a task without following a rigid format <br>
+Assumption : TC002 has been tested and Tester are familiar with the format of adding atask <br>
+
+Steps no. | Steps | Test Data | ExpectedResult
+--- | :---------------- | :---------------- | :----------------
+1 | Add a floating task using the test data | add "very flexible" | the task should be added and highlighted under 'Floating Task' given a default priority of 0.
+2 | Add a deadline | add cs2103Report 10pm | the deadline should be added and highlighted under 'Deadline'
+3 | Add a event | add "picnic with professor" tdy 11pm to tmr | the event should be added and highlighted under 'Event'.
```

Comment:
Event should be added with today 11pm as the start date and tomorrow 23:59 as the end date,